### PR TITLE
ticket 6

### PR DESCRIPTION
**Pull request by Sonia-Heer**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -206,4 +206,47 @@ describe('/api/reviews', () => {
...
+    });
+    test('GET - status: 404 - review_id not found', () => {
+        return request(app)
+        .get('/api/reviews/1000/comments')
+        .expect(404)
+        .then((response) => {
+            expect(response.body.msg).toBe("Not found")
+        });
+    });
+});
```
Comment (robert112233) [inline __tests__/app.test.js:251]: Missing 400 test - /api/reviews/not_an_id/comments

Comment (robert112233) [reply __tests__/app.test.js:251]: Missing 200 test - valid article Id that has no comments, it should return an empty array [] 

You will need a checkReviewExists to differentiate between valid review with no comments (200 db returns empty rows) & invalid review no comments  (404 db also returns empty rows)

**Review by robert112233 — APPROVED** [review]
Include the missing tests  & checkExists and submit another PR for review, great work! 😃

**Review by robert112233 — APPROVED** [review]
Include the missing tests  & checkExists and submit another PR for review, great work! 😃

**Review by robert112233 — APPROVED** [review]
Include the missing tests  & checkExists and submit another PR for review, great work! 😃

**Review by robert112233 — APPROVED** [review]
Include the missing tests  & checkExists and submit another PR for review, great work! 😃

**Review by robert112233 — COMMENTED** [review]
Include the missing tests  & checkExists and submit another PR for review, great work! 😃

**Review by JVal-NC — COMMENTED** [review]
Good work Sonia, go ahead and merge.
